mvn clean install errors

Error is this:
```
$ mvn clean install
...
[ERROR] Failed to execute goal on project ctk-domain: Could not resolve dependencies for project org.ga4gh:ctk-domain:jar:0.6.0a3: Could not find artifact org.apache.bcel:bcel:jar:6.0-SNAPSHOT in Apache (http://repository.apache.org/snapshots) -> [Help 1]
...
```

Apache has possibly stopped hosting the artifact?

--- a/cts-java/src/test/java/org/ga4gh/cts/api/Utils.java
+++ b/cts-java/src/test/java/org/ga4gh/cts/api/Utils.java
@@ -11,7 +11,6 @@ import ga4gh.AlleleAnnotations.VariantAnnotation;
 import ga4gh.AlleleAnnotations.VariantAnnotationSet;
 import ga4gh.Reads.*;
 import ga4gh.ReadServiceOuterClass.*;
-import ga4gh.Reads.ReadGroup.Program;
 import ga4gh.References.*;
 import ga4gh.ReferenceServiceOuterClass.*;
 import ga4gh.SequenceAnnotationServiceOuterClass.SearchFeatureSetsRequest;

--- a/parent/pom.xml
+++ b/parent/pom.xml
@@ -43,7 +43,7 @@
         <assertj.core.version>3.1.0</assertj.core.version>
         <assertj.generator.version>2.0.0</assertj.generator.version>
         <proto.version>3.0.0-beta-3</proto.version>
-        <bcel.version>6.0-SNAPSHOT</bcel.version>
+        <bcel.version>6.0</bcel.version>
         <gson.version>2.3.1</gson.version>
         <guava.version>18.0</guava.version>
         <httpasyncclient.version>4.1</httpasyncclient.version>

